### ISSUE TITLE
Update to Bio-Formats 6.7.0

### ISSUE THREAD

**Issue by imagejan** [Jetraw/Bio-Formats#1]
An up-to-date Fiji ships with `formats-bsd-6.7.0.jar`. The [June 2021 release](https://github.com/Jetraw/Bio-Formats/releases/tag/21.06.16.1) only lists versions 6.5.1, 6.6.0 and 6.6.1.

Is there a way to get the JetRaw codec with an up-to-date Bio-Formats? Alternatively, can the JetRaw codec be distributed as a separate package? Or maybe as a SCIFIO reader plugin? This would simplify its usage in a modular environment.

**Comment by arvequina** [comment]
@imagejan we will provide an update to 6.7.0 in the coming days. I know it is not ideal but in the meantime you can use previous versions of Fiji. The latest Bio-Formats was only released 2/3 weeks ago.  

We also offer a separate package to decompress Jetraw images without Bio-Formats:
https://github.com/Jetraw/Fiji

The main problem is that many users need the capabilities that Bio-Formats offer, specially regarding all the information stored in the metadata.  

Regarding SCIFIO, I do not know much about it but it might be worth exploring this option. Thanks!

**Comment by imagejan** [comment]
Thanks for the info, @arvequina!

The advantage with SCIFIO would be that you can support the codec in its own class, e.g as it's done in [`io.scif.codec.JPEGCodec`](https://github.com/scifio/scifio/blob/e0dcf44d7062da558ca3aeee44f2b45cedddd584/src/main/java/io/scif/codec/JPEGCodec.java#L53-L54).

```
@Plugin(type = Codec.class)
public class JetRawCodec extends AbstractCodec {
  ...
}
```

SCIFIO's extensibility would allow for a better modularity, without the need of shipping a library that shades another one from Fiji's core update sites.

Alternatively, I propose shipping the modified bio-formats libraries via an [update site](https://imagej.net/update-sites/index) so you can make sure you distribute a compatible set of Bio-Formats components.

**Comment by arvequina** [comment]
Thanks to you @imagejan 

I was taking a look at the SCIFIO project and you are right that it could be quite convenient for us. I did not know about its existence. 

Regarding the update site at the moment we are approaching one by one all our clients but it is true that for the future is a much better way (and the proper one) to inform everyone at once.

Thanks again for your contribution!

**Comment by arvequina** [comment]
@imagejan just published the up-to-date version for Bio-Formats 6.7.0

We will discuss internally if we move to SCIFIO (thanks for proposing it.)

Regarding how to ship the updates we have our website where you will always find the latest versions of the software and the corresponding plug-ins:
https://jetraw.com/downloads/software

The website redirects you to this github repository but you have a clear summary of all plug-ins and current versions. 

With your permission I will proceed to close the issue.
